module/Activity/src/Activity/Mapper/Activity.php

```diff
@@ -79,12 +94,27 @@ public function getUpcomingActivities($count = null)
      *
      * @return array
      */
-    public function getAllApproved()
+    public function getApprovedActivities()
```

Comment:
Because of DRY, is it not better to make one method for this, `findActivitiesByStatus`?